prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+alert('connected');
```

Comment:
In future exercises be sure to remove lines of code like this that are no longer relevant to the exercise :)